tests/test_livyclient.py

```diff
@@ -122,10 +131,58 @@ def test_get_logs_returns_session_logs():
 def test_get_logs_returns_false_with_value_error():
     err = "err"
     mock_spark_session = MagicMock()
-    type(mock_spark_session).logs = PropertyMock(side_effect=ValueError(err))
+    mock_spark_session.get_logs = MagicMock(side_effect=ValueError(err))
     client = LivyClient(mock_spark_session)
 
     res, logs_r = client.get_logs()
 
     assert not res
     assert logs_r == err
+
+
+@with_setup(_setup, _teardown)
+def test_execute_sql():
+    sqlquery = SQLQuery("HERE IS THE QUERY", "take", 100, 0.2)
```

Comment:
Should we mock SQLQuery on these class's unit tests? I'm worried the implementation of the `to_command` method could impact the unit tests here. Should we decouple?